### ISSUE TITLE
Audit dependencies

### ISSUE THREAD

**Issue by pdehaan** [mozilla/something-awesome#81]
I use [**depcheck**](http://npm.im/depcheck), but looks like we have a couple unused dependencies:

```sh
$ depcheck

Unused dependencies
* color-convert
* fs-extra
* path

Unused devDependencies
* browserify
* buffer-shims
* eslint-plugin-node
* eslint-plugin-security
* stylelint-config-standard
```

1. **path** is a native Node module, and needn't be in package.json. I'm not actually sure what package we're randomly installing from npm, but always kinda weird that it "works".
1. We can ignore **browserify**, __eslint-plugin-*__, and __stylelint-*__ since those are used in npm scripts and are red herrings.

---

**TL;DR:** Not sure if we still need **color-convert**, **fs-extra**, or **buffer-shims** in the package.json [dev]dependencies anymore. ~~We definitely don't need an npm installed **path** module.~~ (Removed in #104)
